### ISSUE TITLE
Dropdown, onChange event is not firing in NON react context

### ISSUE THREAD

**Issue by LucaMele** [axa-ch/patterns-library#1208]
in dropdown src code, handleDropdownItemClick **is triggered** when changing event on line 196 in patterns-library/src/components/20-molecules/dropdown/index.js

Console log here is not fireing instead -> see onchange="console.log"
<img width="1060" alt="Bildschirmfoto 2019-08-06 um 10 13 48" src="https://user-images.githubusercontent.com/275235/62522902-ed435800-b832-11e9-869c-555867e2062b.png">

UPDATE:
Also adding an event listsner wont work like so due to e.stopPropagation() in `handleDropdownItemClick()`:

```
  <axa-dropdown
                  class="js-hack"
                  data-sly-use.plibHelper="${'com.axa.ch.aem.sites.slingmodels.PatternsLibraryDataConversionHelper' @ xssAPI=xssAPI}"
                  title="${properties.labelselection}"
                  onchangeeee="controlClick(event.target.options[event.target.selectedIndex].getAttribute('data-url'), event.target.value)"
                  onchange="console.log(this)"
                  items="${plibHelper.items}">
          </axa-dropdown>
          <script>
            document.querySelector('.js-hack').addEventListener('change', (e) => {console.log(e)});
            document.querySelector('.js-hack').addEventListener('onchange', (e) => {console.log(e)});
            document.querySelector('.js-hack').addEventListener('onChange', (e) => {console.log(e)});
          </script>

```

**Comment by markus-walther** [comment]
For posteriority, the root problem was that inlined native event handlers like `onchange` cannot be converted into callable functions by lit-element.

The fix is to synthetically trigger a custom `change` event with suitable event details on the component instance itself.

**Comment by LucaMele** [comment]
fixed with #1210
